### DISCUSSION TITLE
Use of mkdocs (as mentioned in podcast.__init__)?

### DISCUSSION THREAD

**Discussion by gimbo** [pyscaffold/pyscaffold#739] [Q&A]
I heard about pyscaffold via [the recent episode of Podcast.\_\_init\_\_](https://www.pythonpodcast.com/pyscaffold-project-template-boilerplate-episode-384) in which Florian Wilhelm was interviewed.

He mentioned in that interview that the pyscaffold community is currently evaluating moving from sphinx to mkdocs (if I understood correctly) for documentation generation, and as someone who's used cookiecutter a bit and who likes mkdocs, my curiosity was piqued.

However, I don't see anything about mkdocs in project documentation, branches, issues, PRs, extensions, or this GH discussion section (other than cursory mentions in some discussions and one PR).

I just wanted to ask what the status of this is... Is the discussion happening elsewhere (a Discord?), say? And is the transition likely to happen in the near future?

**Comment by abravalheri** [comment]
Hi @gimbo right now we are still using sphinx for markdown docs.

We haven't planned this transition yet.

The big advantage of sphinx is that it allows people to use either restructuredtext or markdown. That can be quite handy for PyScaffold.
